src/ansys/fluent/core/meta.py

```diff
@@ -149,10 +149,10 @@ def __create_get_ancestors_by_type(cls):
         def wrapper(self, obj_type, obj=None):
             obj = self if obj is None else obj
             parent = None
-            if getattr(obj, "_parent", None):
-                if isinstance(obj._parent, obj_type):
-                    return obj._parent
-                parent = self.get_ancestors_by_type(obj_type, obj._parent)
+            if getattr(obj, "parent", None):
```

Comment:
We also want a method to avoid writing `parent.parent.parent.parent.parent`. Do we have that here?

Reply:
yes, this is that method. @seanpearsonuk